### PR TITLE
added skeleton loading

### PR DESCRIPTION
**Pull request by hasnain-cyber**
added the skeleton loading in home page file, and also added some dummy data in the same file to test the feature.

### CODE REVIEW

File: client/src/pages/public/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -3,26 +3,74 @@ import Navbar from '../../../components/public/Navbar/Navbar';
 import Hero from '../../../components/public/Hero/Hero';
 import Footer from '../../../components/public/Footer/Footer';
 import Section from '../../../components/public/Section/Section';
+import { Box, Skeleton } from '@mui/material'
+import './HomePage.css'
 
+// const sections = [
```
Comment (akashg3627) [inline client/src/pages/public/HomePage/HomePage.jsx:9]: where from this came?

File: client/src/pages/public/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -3,26 +3,74 @@ import Navbar from '../../../components/public/Navbar/Navbar';
...
-        <Footer/>
+      {/*footer*/}
+      <Footer />
 
     </div>
   )
 }
+
+
+const SkeletonLoadingTemplate = () => {
```
Comment (akashg3627) [inline client/src/pages/public/HomePage/HomePage.jsx:68]: make it as different component in new file

File: client/src/pages/public/HomePage/HomePage.jsx [hunk 3]
```diff
@@ -3,26 +3,74 @@ import Navbar from '../../../components/public/Navbar/Navbar';
...
+      <Hero />
 
 
-        {/*body*/}
-        {
-            sections.map(section=><Section key={section.sectionID} section={section}/>)
-        }
+      {/*body*/}
+      {sections.length > 0 ? sections.map(section => <Section key={section.sectionID} section={section} />) :
+        <Box display='flex' flexDirection={'column'} alignItems={'center'}>
```
Comment (akashg3627) [inline client/src/pages/public/HomePage/HomePage.jsx:52]: combine this box into another component

File: client/src/pages/public/HomePage/HomePage.jsx (hunk 1 above)
Comment (hasnain-cyber) [reply client/src/pages/public/HomePage/HomePage.jsx:9]: Bhaiya, this is just dummy data which we can use during testing phase as actual data is not available currently.

File: client/src/pages/public/HomePage/HomePage.jsx (hunk 2 above)
Comment (hasnain-cyber) [reply client/src/pages/public/HomePage/HomePage.jsx:68]: Ok bhaiya 👍

File: client/src/pages/public/HomePage/HomePage.jsx (hunk 3 above)
Comment (hasnain-cyber) [reply client/src/pages/public/HomePage/HomePage.jsx:52]: Ok bhaiya 👍

**Review by akashg3627 — APPROVED** [review]

**Pull request merged**